cmd/juju/status/output_tabular.go

```diff
@@ -177,6 +177,9 @@ func FormatTabular(writer io.Writer, forceColor bool, value interface{}) error {
 		if agentDoing != "" {
 			message = fmt.Sprintf("(%s) %s", agentDoing, message)
 		}
+		if u.Leader {
```

Comment:
There is no nice way to not do this. Discussed on the bug with sabdfl, and agreed that at least for the first cut, we always show it.